README.md

```diff
@@ -1,2 +1,51 @@
 # robotont_demos
 This repository is a ROS package that contains various demos showing the capabilities of the Robotont platform
+
```

Comment:
Oleks vaja sellist alapeatükki, nagu "Setup" või "Before you begin". Midagi, kus on kirjas, et demode jaoks on tarvis nt 1 ühte füüsilist robotit. ja ühe PC-d, kuhu on paigaldatud Ubuntu Linux ja ROS Melodic. Mina soovitan robotondi arvutit nimetada läbivalt kui Robotont's _on-board computer_ ja seda kasutaja arvutit nt _user PC_. Siis on alati võimalik selgelt öelda, millises arvutis, mis käsku jooksutada. Ühtlasi võiks lisada lause: kui demo toimib ka simuleeritud juhul, siis on vastava demo alajaotises see kirjeldatud.